nixos/modules/services/mail/maddy.nix

```diff
@@ -335,12 +335,13 @@ in {
       };
 
       secrets = lib.mkOption {
-        type = lib.types.path;
+        type = with types; nullOr path;
```

Comment:
I'd either name it singular secret and only support one secrets file (like it's currently implemented) or make it plural and of type `listOf path` (`serviceConfig.EnvironmentFile` happily accepts a list). I personally would prefer the second option.